### PR TITLE
Fix contracts 0.8.9 build for test:fix on win platform

### PR DESCRIPTION
**Pull request by StefanoGagliardi**
Hi guys,
during set up for Development i try to run "test:fix" on window and get error about "contracts not found" for 0.8.9.
Fix is add base path to solc contract compile process.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -4,15 +4,28 @@ We welcome all contributions!
 
 ## Developing
 
-Heads up: we use pnpm instead of yarn or npm. It might be new for you but I promise it's worth the hassle - it's really
-good at making big monorepos deterministic. Please ensure that you are running pnpm version 7.
+Heads up: we use pnpm instead of yarn or npm. It might be Q new for you but I promise it's worth the hassle - it's
```
Comment (krzkaczor) [inline CONTRIBUTING.md:7]: `Q` typo

File: contracts/v0.6.4/LibraryPerson.sol [hunk 2]
```diff
@@ -0,0 +1,48 @@
+pragma solidity ^0.6.4;
```
Comment (krzkaczor) [inline contracts/v0.6.4/LibraryPerson.sol:1]: can we move these contracts to a separate PR? lets just fix windows development in this one

**Review by krzkaczor — CHANGES_REQUESTED** [review]

File: contracts/v0.6.4/LibraryPerson.sol (hunk 2 above)
Comment (StefanoGagliardi) [reply contracts/v0.6.4/LibraryPerson.sol:1]: Absolutely, sorry I didn't realize it had taken multiple commits.

Comment (krzkaczor) [reply contracts/v0.6.4/LibraryPerson.sol:1]: Will you find time to fix it or should I step in? :)

Comment (StefanoGagliardi) [reply contracts/v0.6.4/LibraryPerson.sol]: Absolutely, I just removed the files, sorry again, I was convinced I was committing to my fork.
The PR idea was just for the "compile-contract.ts" file.
Contributing.md are my personal notes, do we keep them or do you prefer to remove them?

Thanks

Comment (krzkaczor) [reply contracts/v0.6.4/LibraryPerson.sol]: Thanks!

**Review by krzkaczor — APPROVED** [review]

**Comment by krzkaczor** [comment]
Awesome, thank you!
